oam/install_scripts/mariadb-columnstore-start.sh

```diff
@@ -2,14 +2,21 @@
 
 # This script allows to gracefully start MCS
 
+# prevent nodes using shared storage manager from stepping on each other when initializing
+# flock will open up an exclusive file lock to run atomic operations
+exec {fd_lock}>/var/lib/columnstore/storagemanager/storagemanager-lock
```

Comment:
I wonder what does this bash syntax do?

Reply:
it will create a file descriptor `fd_lock` pointing to storagemanager-lock, which is then used with `flock -n $fd_lock `(takes in a file descriptor) to provide atomic operations